.github/workflows/CI.yml

```
@@ -20,7 +20,7 @@ jobs:
     strategy:
       matrix:
         version:
-          - '1.6'
+          - '1.8'
```

This is the same as '1' (see https://github.com/JuliaOptimalTransport/PythonOT.jl/pull/21). IMO it's important to test the oldest Julia version that we want to support, the stable version, and possibly if not included already Julia LTS. So I suggest reverting this change:
```suggestion
          - '1.6'
```